runtime/vara/src/lib.rs

```diff
@@ -415,100 +588,40 @@ pallet_staking_reward_curve::build! {
 
 // TODO: review staking parameters - currently copying Kusama
 parameter_types! {
-    // Six sessions in an era (1 hour)
+    // Six sessions in an era (4 hours)
     pub const SessionsPerEra: sp_staking::SessionIndex = 6;
-    // In production: 28 * 24 eras for unbonding (7 days)
-    // TODO: should we require bonding duration in stage-1?
-    pub const BondingDuration: sp_staking::EraIndex = 0;
-    // 1/4 the bonding duration (in prod 24 * 7)
-    pub const SlashDeferDuration: sp_staking::EraIndex = 0;
+    // 28 * 6 eras for unbonding (28 days)
+    pub const BondingDuration: sp_staking::EraIndex = 6 * 28;
+    // 27 days (in eras) during which slashes can be cancelled
+    pub const SlashDeferDuration: sp_staking::EraIndex = 6 * 27;
```

Comment:
6 eras eq 4 hours
What about introducing some constant like eras per day?

Reply:
Haven't seen any other network doing it (Substrate, Polkadot etc). But of course we can introduce constants to avoid confusion.